.flowconfig

```diff
@@ -1,7 +1,7 @@
 [ignore]
 .*/src/api/.*
 .*/src/core/.*
-.*/dist/.*
+.*/(?<!ripple-keypairs/)dist/.*
```

Comment:
That means we might run into this confusing issue again though, it's hard to know if/when the flow team will fix it: https://github.com/facebook/flow/issues/510

Isn't deleting the "dist" line a better option for now?